internal/typeinfo/arginfo_test.go

```diff
@@ -61,10 +61,15 @@ func (s *typeInfoSuite) TestArgInfoStruct(c *C) {
 		tag:       "name",
 	}}
 
-	allOutputs, names, err := argInfo.AllStructOutputs("myStruct")
+	allOutputs, outputNames, err := argInfo.AllStructOutputs("myStruct")
 	c.Assert(err, IsNil)
 	c.Assert(allOutputs, HasLen, len(structFields))
 
+	allInputs, inputNames, err := argInfo.AllStructInputs("myStruct")
+	c.Assert(err, IsNil)
+	c.Assert(allInputs, HasLen, len(structFields))
```

Comment:
After you have asserted a nil error, which you want to fail immediately to prevent later check panics, prefer to use `c.Check`, which does not jump out of the test immediately. That way you get more feedback for failure cases.

Reply:
Changed.